### PR TITLE
Add reference to Armeria starter

### PR DESCRIPTION
**Pull request by jaeyeonling**
This PR adds a link to the Armeria to the list of Spring Boot Starters.

Thanks!

### CODE REVIEW

File: spring-boot-project/spring-boot-starters/README.adoc [hunk 1]
```diff
@@ -205,4 +205,7 @@ do as they were designed before this was clarified.
 | https://alexo.github.io/wro4j/[Wro4j]
 | https://github.com/michael-simons/wro4j-spring-boot-starter
 
+| https://line.github.io/armeria/[Armeria]
```
Comment (snicoll) [inline spring-boot-project/spring-boot-starters/README.adoc]: This file is ordered alphabetically, can you please move this entry accordingly?

Comment (jaeyeonling) [reply spring-boot-project/spring-boot-starters/README.adoc]: I fixed that :)

**Comment by snicoll** [comment]
@jaeyeonling thanks for following up and for making your first contribution to Spring Boot.

**Comment by trustin** [comment]
Thanks for adding us, @jaeyeonling and @snicoll :heart: 
/cc @minwoox @ikhoon

**Pull request closed without merging**